code/__HELPERS/_lists.dm

```diff
@@ -11,20 +11,24 @@
 
 #define LAZYINITLIST(L) if (!L) L = list()
 #define UNSETEMPTY(L) if (L && !length(L)) L = null
-#define LAZYCOPY(L) (L ? L.Copy() : list() )
+#define LAZYLISTDUPLICATE(L) (L ? L.Copy() : null )
```

Comment:
Document it.

Reply:
`///Like LAZYCOPY - copies an input list if the list has entries, If it doesn't the assigned list is nulled`